docker-entrypoint.sh

```diff
@@ -19,6 +19,35 @@ run_as() {
     fi
 }
 
+# Execute all executable files in a given directory in alphanumeric order
+run_path() {
+    local hook_folder_path="/docker-entrypoint-hooks.d/$1"
+    local return_code=0
+
+    echo "=> Searching for scripts (*.sh) to run in the folder: ${hook_folder_path}"
+
+    (
+        cd "${hook_folder_path}"
+        find . -type f -iname '*.sh' -print | sort | while read script_file_path; do
```

Comment:
The idea was that people could make folders, as a way to categorize their scripts, and `find` would be able to locate them 🙂

But if you don't like the idea, I have no problem changing it to
```
cd "${hook_folder_path}"
for f in ./*.sh; do
```
Because I already use `cd`, I don't need `$hook_folder_path/` and the `*` (asterisk) does not work if it is wrapped in `"` (double quote)

Reply:
Hey @J0WI

I have done as you requested
- `cd` is removed
- `find` is replace with `for script_file_path in "${hook_folder_path}/"*.sh; do`

I also updated the README.md file to specify that scripts in sub-folders of the hook-folders will not be executed. And, when running a script it will also print the current working directory (I assume this info could be helpful)